<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataPusherWidget</class>
 <widget class="QWidget" name="DataPusherWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="topButtonHolderWidget" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="quickPlotSetupWidget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QScrollArea" name="leftScrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="leftScrollAreaWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>151</width>
           <height>264</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="leftVerticalLayout"/>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="centralButtonsWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="rightArrowPushButton">
           <property name="toolTip">
            <string>Create track for log type</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deletePushButton">
           <property name="toolTip">
            <string>Delete selected track</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="upPushButton">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>Promote track (moves towards left in plot)</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="downPushButton">
           <property name="toolTip">
            <string>Demote track (moves towards right in plot)</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="rightScrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="rightScrollAreaWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>151</width>
           <height>264</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="rightVerticalLayout"/>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
